test/gst-msdk/encode/av1.py

```diff
@@ -102,3 +116,9 @@ def init(self, tspec, case, gop, bframes, tilecols, tilerows, bitrate, fps, qual
   def test(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
+
+  @slash.parametrize(*gen_av1_vbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```

Comment:
`spec` -> `spec_r2r`

Reply:
thanks review and catch this issue, updated